packages/raja/package.py

```diff
@@ -192,11 +205,12 @@ class Raja(CachedCMakePackage, CudaPackage, ROCmPackage):
     depends_on("blt@0.3.6:0.4.1", type="build", when="@:0.12.0")
     conflicts("^blt@:0.3.6", when="+rocm")
 
+    depends_on("camp",type="build")
```

Comment:
Why is the type set to "build"?

Reply:
I got that from @white238. I think it's just a generic way of saying that camp is part of RAJA's build stack.

Reply:
I thought that was a way of marking dependencies that aren't libraries (e.g. cmake, blt).

Reply:
Could be a relic from when camp was header only?

Reply:
The types are `build`, `link`, `run`, and `test`. `build` by itself is not correct here.

Reply:
So replace with ``depends_on("camp")`` ??

Reply:
Fixed.

Reply:
Thank you @adayton1